### ISSUE TITLE
Add languages thing

### ISSUE THREAD

**Issue by Aldair502** [taisei-project/taisei#376]
Idk

**Comment by Nhalrath** [comment]
A language selector? I think it's a nice addition to the game, but do we have people willing to translate the game?

I'm no professional in programming and Taisei's large codebase still overwhelms me, but maybe I can give this one a shot! It's a good opportunity to learn more about the internals of the game.

**Comment by Nhalrath** [comment]
I attempted to take this one today, but it looks like non-English characters are not supported. My guess is that the fonts themselves doesn't have the necessary glyphs for these.
![screenshot](https://github.com/taisei-project/taisei/assets/69453927/0db3d8d9-27c3-4747-8266-9a53914563dd)

I'm not familiar with how fonts are handled, so I guess I'll pass on this one.

**Comment by Firestar-Reimu** [comment]
Yes, I wish to make a Chinese translation **(as non-CJK characters are too out-of-character when playing this game)**, and the Chinese/Japanese font can use the open source `Source Han Sans`.
